package.json

```diff
@@ -58,10 +58,12 @@
   "dependencies": {
     "@plotly/d3-sankey": "0.7.2",
     "@plotly/d3-sankey-circular": "0.33.1",
+    "@plotly/point-cluster": "^3.1.9",
     "@turf/area": "^6.0.1",
     "@turf/bbox": "^6.0.1",
     "@turf/centroid": "^6.0.2",
     "alpha-shape": "^1.0.0",
+    "bubleify": "^2.0.0",
```

Comment:
This is in `dependencies`, did you mean to put it in `devDependencies`?

Reply:
@andersk Good eye! Possibly misspelled the `--save-dev` while running the `npm install` command.
Fixed in https://github.com/plotly/plotly.js/pull/5084/commits/b7e16f409f16b5e515e415dc6f56dd6ab56fa547.